crates/sui-types/src/error.rs

```diff
@@ -447,6 +447,8 @@ pub enum SuiError {
         new_epoch: EpochId,
         locked_by_tx: TransactionDigest,
     },
+    #[error("Too many requests")]
+    TooManyRequests,
```

Comment:
lets add new errors at the end to avoid breaking deserialization for old clients